index.js

```diff
@@ -1,87 +1,109 @@
-require(__dirname + '/js/Track/library/dbSNP.js');
-require(__dirname + '/js/Track/library/File.js');
-require(__dirname + '/js/Track/library/File/BAM.js');
-require(__dirname + '/js/Track/library/File/BED.js');
-require(__dirname + '/js/Track/library/File/BIGBED.js');
-require(__dirname + '/js/Track/library/File/BIGWIG.js');
-require(__dirname + '/js/Track/library/File/GFF.js');
-require(__dirname + '/js/Track/library/File/VCF.js');
-require(__dirname + '/js/Track/library/File/WIG.js');
-require(__dirname + '/js/Track/library/Gene.js');
-require(__dirname + '/js/Track/library/HighlightRegion.js');
-require(__dirname + '/js/Track/library/Legend.js');
-require(__dirname + '/js/Track/library/Scaleline.js');
-require(__dirname + '/js/Track/library/Scalebar.js');
+require('./js/lib/jquery.mousewheel.js');
+require('./js/lib/jquery.mousehold.js');
+require('./js/lib/jquery.tipsy.js');
+
+const Genoverse = require('./js/Genoverse.js');
+const { Track, Static, Legend }  = require('./js/Track.js');
+Genoverse.Track = Track;
+
+Genoverse.Track.Controller = require('./js/Track/Controller.js');
+Genoverse.Track.Model = require('./js/Track/Model.js');
+Genoverse.Track.View = require('./js/Track/View.js');
+
+Genoverse.Track.Static = Static.Track;
+Genoverse.Track.View.Static = Static.View;
+Genoverse.Track.Model.Static = Static.Model;
+Genoverse.Track.Controller.Static = Static.Controller;
+
+Genoverse.Track.Controller.Stranded = require('./js/Track/Controller/Stranded.js');
+Genoverse.Track.Model.Stranded = require('./js/Track/Model/Stranded.js');
+
+const Graph = require('./js/Track/library/Graph.js');
+Genoverse.Track.Graph = Graph.Track;
+Genoverse.Track.View.Graph = Graph.View;
+Genoverse.Track.Model.Graph = Graph.Model;
+Genoverse.Track.Controller.Graph = Graph.Controller;
+
+const Line = require('./js/Track/library/Graph/Line.js');
+Genoverse.Track.Graph.Line = Line.Track;
+Genoverse.Track.View.Graph.Line = Line.View;
+Genoverse.Track.Model.Graph.Line = Line.Model;
+Genoverse.Track.Controller.Graph.Line = Line.Controller;
+
+const Bar = require('./js/Track/library/Graph/Bar.js'); // Graph.Bar depends on Graph.Line
+Genoverse.Track.Graph.Bar = Bar.Track;
+Genoverse.Track.View.Graph.Bar = Bar.View;
+Genoverse.Track.Model.Graph.Bar = Bar.Model;
+Genoverse.Track.Controller.Graph.Bar = Bar.Controller;
+
+Genoverse.Track.Controller.Sequence = require('./js/Track/Controller/Sequence.js');
+Genoverse.Track.Model.Sequence = require('./js/Track/Model/Sequence.js');
+Genoverse.Track.Model.Sequence.Fasta = require('./js/Track/Model/Sequence/Fasta.js');
+Genoverse.Track.Model.Sequence.Ensembl = require('./js/Track/Model/Sequence/Ensembl.js');
+Genoverse.Track.View.Sequence = require('./js/Track/View/Sequence.js');
+Genoverse.Track.View.Sequence.SequenceVariation = require('./js/Track/View/Sequence/Variation.js');
+
+Genoverse.Track.Model.Sequence.SequenceVariation = require('./js/Track/Model/SequenceVariation.js');
```

Comment:
I found a problem here when I was looking at getting the tests running on this branch - this should be...:

```suggestion
Genoverse.Track.View.SequenceVariation = require('./js/Track/View/Sequence/Variation.js');

Genoverse.Track.Model.SequenceVariation = require('./js/Track/Model/SequenceVariation.js');
```